### PR TITLE
BETA: Register zeus-sec.is-a.dev

### PR DESCRIPTION
**Pull request by l33t1**
Added `zeus-sec.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/x1337.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "l33t1",
+        "email": "yassineonet@gmail.com"
+    },
+    "record": {
+        "A": ["178.33.36.88"]
```
Comment (Stef-00012) [inline domains/x1337.json:7]: I cannot reach your website, 404

File: domains/x1337.pw.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "l33t1",
+        "email": "yassineonet@gmail.com"
+    },
+    "record": {
+        "A": ["185.234.67.28"]
```
Comment (Stef-00012) [inline domains/x1337.pw.json:7]: I cannot reach your website, 404

File: domains/zeusapi.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "l33t1",
+        "email": "yassineonet@gmail.com"
+    },
+    "record": {
+        "A": ["185.234.67.28"]
```
Comment (Stef-00012) [inline domains/zeusapi.json:7]: I cannot reach your website, 404

File: domains/legit.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "l33t1",
+        "email": "yassineonet@gmail.com"
+    },
+    "record": {
+        "URL": "https://zeusapi.ch"
```
Comment (Stef-00012) [inline domains/legit.json:7]: I cannot reach your website, Connection timed out Error code 522

**Review by Stef-00012 — CHANGES_REQUESTED** [review]

**Review by l33t1 — COMMENTED** [review]
{
    "description": "Describe the use of this subdomain",
    "repo": "https://github.com/l33t1/l33t1.github.io",
    "owner": {
        "username": "l33t1",
        "email": "yassineonet@gmail.com",
        "twitter": "<twitter-username>"
    },
    "record": {
        "CNAME": "l33t1.github.io"
    }
}

**Comment by triach-rold** [comment]
Website is still unreachable, btw. (Seems like the repo has been deleted too.)

**Pull request closed without merging**